depends/Makefile

```diff
@@ -89,6 +90,7 @@ full_host_os:=$(subst $(host_arch)-$(host_vendor)-,,$(canonical_host))
 host_os:=$(findstring linux,$(full_host_os))
 host_os+=$(findstring darwin,$(full_host_os))
 host_os+=$(findstring freebsd,$(full_host_os))
+host_os+=$(findstring netbsd,$(full_build_os))
```

Comment:
`full_host_os` :)

Reply:
Addressed.

Reply:
Does it still build after this change? I guess you've been testing in a VM where the build and host are the same?

Reply:
> Does it still build after this change?

Yes.

> I guess you've been testing in a VM where the build and host are the same?

I haven't been using a VM. I've been testing using NetBSD 9.99 running on aarch64 hardware.